feature-libs/user/profile/components/user-profile-components.module.ts

```diff
@@ -22,6 +23,7 @@ import { UpdateProfileModule } from './update-profile/update-profile.module';
     ForgotPasswordModule,
     ResetPasswordModule,
     CloseAccountModule,
+    AddressBookModule,
```

Comment:
I wonder if this is the correct place to put `AddressBookModule`. The rest of the modules, match with their functionalities in one way or another. Also, I'm not sure if it matters but now when you load the address form it loads the whole profile module (update profile, register, etc.).

Reply:
I moved to that location because the userProfileFacade was present. The objective was to eliminate UserService and utilize userProfileFacade.